exception-handling.md

```diff
@@ -0,0 +1 @@
+* [Exception Handling in Java](https://www.enjoyalgorithms.com/blog/exception-handling-java) - enjoy algorithms
```

Comment:
Please move the resource to proper file. Read [Contributing.md](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md) for more information